woocommerce-reset.php

```diff
@@ -240,3 +259,31 @@ function schedule_event( $hook, $args = array() ) {
 	uksort( $crons, 'strnatcasecmp' );
 	return _set_cron_array( $crons );
 }
+
+/**
+ * Switch site language.
+ *
+ * @param $request
+ * @return void
+ */
+function switch_language($request) {
+	$lang = $request->get_param('lang');
+	if ($lang !== 'en_US') {
+		$wp_lang_dir = ABSPATH . '/wp-content/languages';
+
+		$core_lang = $lang . '.mo';
+		$admin_lang = 'admin-' . $lang . '.mo';
+		$plugin_lang = 'woocommerce-es_ES.mo';
+
+		if (!is_dir($wp_lang_dir . '/plugins')) {
+			mkdir($wp_lang_dir . '/plugins');
```

Comment:
I encountered errors in my JN site:

```
<br />
<b>Warning</b>:  mkdir(): No such file or directory in <b>/srv/users/user442e25a8/apps/user442e25a8/public/wp-content/plugins/woocommerce-reset/woocommerce-reset.php</b> on line <b>279</b><br />
<br />
<b>Warning</b>:  copy(/srv/users/user442e25a8/apps/user442e25a8/public//wp-content/languages/es_ES.mo): failed to open stream: No such file or directory in <b>/srv/users/user442e25a8/apps/user442e25a8/public/wp-content/plugins/woocommerce-reset/woocommerce-reset.php</b> on line <b>282</b><br />
<br />
<b>Warning</b>:  copy(/srv/users/user442e25a8/apps/user442e25a8/public//wp-content/languages/admin-es_ES.mo): failed to open stream: No such file or directory in <b>/srv/users/user442e25a8/apps/user442e25a8/public/wp-content/plugins/woocommerce-reset/woocommerce-reset.php</b> on line <b>283</b><br />
<br />
<b>Warning</b>:  copy(/srv/users/user442e25a8/apps/user442e25a8/public//wp-content/languages/plugins/woocommerce-es_ES.mo): failed to open stream: No such file or directory in <b>/srv/users/user442e25a8/apps/user442e25a8/public/wp-content/plugins/woocommerce-reset/woocommerce-reset.php</b> on line <b>284</b><br />
<br />
<b>Warning</b>:  Cannot modify header information - headers already sent by (output started at /srv/users/user442e25a8/apps/user442e25a8/public/wp-content/plugins/woocommerce-reset/woocommerce-reset.php:279) in <b>/srv/users/user442e25a8/apps/user442e25a8/public/wp-includes/rest-api/class-wp-rest-server.php</b> on line <b>1717</b><br />
true
```

I think we need to create `$wp_lang_dir` if it does not exist.

Reply:
Thank you for catching it! I'll make an update.

Reply:
@chihsuan Updated in https://github.com/woocommerce/woocommerce-reset/pull/8/commits/5755c4bd3bfdd12cd8bd7baec47f2500524de000